pandas/tests/series/test_operators.py

```diff
@@ -595,77 +595,103 @@ def test_divide_decimal(self):
 
         assert_series_equal(expected, s)
 
-    def test_div(self):
+    def test_i8_ser_div_i8_ser(self):
+        # no longer do integer div for any ops, but deal with the 0's
+        first = Series([3, 4, 5, 8], name='first')
+        second = Series([0, 0, 0, 3], name='second')
+
         with np.errstate(all='ignore'):
-            # no longer do integer div for any ops, but deal with the 0's
-            p = DataFrame({'first': [3, 4, 5, 8], 'second': [0, 0, 0, 3]})
-            result = p['first'] / p['second']
-            expected = Series(
-                p['first'].values.astype(float) / p['second'].values,
-                dtype='float64')
-            expected.iloc[0:3] = np.inf
-            assert_series_equal(result, expected)
+            expected = Series(first.values.astype(np.float64) / second.values,
+                              dtype='float64')
+        expected.iloc[0:3] = np.inf
 
-            result = p['first'] / 0
-            expected = Series(np.inf, index=p.index, name='first')
-            assert_series_equal(result, expected)
+        result = first / second
+        assert_series_equal(result, expected)
 
-            p = p.astype('float64')
-            result = p['first'] / p['second']
-            expected = Series(p['first'].values / p['second'].values)
-            assert_series_equal(result, expected)
+    def test_f8_ser_div_f8_ser(self):
```

Comment:
same as above, maybe make a fixture for float32/float64 (could put at in pandas/tests/conftest.py) and for ints/uints as well. I think there is an issue about doing this.